docker-compose.yml

```diff
@@ -560,7 +560,7 @@ services:
       environment:
         - PUBSUB_PROJECT1=testproject,topic1:subscription1
     loki:
-      image: grafana/loki
+      image: grafana/loki:master-89263b0-amd64
```

Comment:
can we set this in `docker-compose.yml` as well?

Reply:
I tried to run `grep -R 'grafana/loki' .` in Vector's source directory and it was the only place where this Docker image was used.

Reply:
Oh sorry, I haven't had my coffee yet I though this was circleci's config :) sorry about that